daemon.lua

```diff
@@ -37,17 +46,23 @@ end
 local function daemon()
 	print("[ring-0/dome] started")
 	while (not shouldstop()) do
-		local message = inbox:receive()
+		local message, frame = inbox:receive()
 		if message then
 			telegraf:push(header .. message)
-		else
+		end
+		if frame then
+			local packet = data.new(#frame)
+			packet:setstring(0, frame)
+			reply.https(packet)
```

Comment:
I think we shouldn't tie this API to HTTPS.. it's a TCP RST.. I would call it `reply.rst` or `reply.reset` or `reply.tcp_rst`, etc